<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>733</width>
    <height>441</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>733</width>
    <height>441</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>733</width>
    <height>441</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>txt to access for TPTS</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="button_start">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>260</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_dir_selects">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Обзор</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_dir_db">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Обзор</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget_log">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>290</y>
      <width>731</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>551</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Выберете папку с исходными данными, который хотите перенести в базу данных MS Access</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_dir_db">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>431</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="buddy">
     <cstring>button_dir_db</cstring>
    </property>
   </widget>
   <widget class="QLabel" name="label_dir_selects">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>80</y>
      <width>431</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="buddy">
     <cstring>button_dir_selects</cstring>
    </property>
   </widget>
   <widget class="QLabel" name="label_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>491</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Выберете  базу данных MS Access в формате accdb, в которую будут записаны данные</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_clear_db">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Очистить</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>361</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Очистить базу от таблиц</string>
    </property>
    <property name="buddy">
     <cstring>button_clear_db</cstring>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_EM">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>220</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>EM</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_NT">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>220</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>HT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>260</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>i</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>733</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
